CHANGELOG.md

```diff
@@ -11,17 +11,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-- `U768` alias ([#310])
 - `bytemuck` feature ([#292])
 - `Uint::is_zero() -> bool` ([#296])
 - `num-traits` features ([#298])
+- `U768` alias ([#310])
```

Comment:
looks like i forgot to add the link to the block below. mind doing that as a drive-by?